### ISSUE TITLE
Bug

### ISSUE THREAD

**Issue by MysterousPotato** [Freenex1911/FeexRanks#3]
Hi, im new n' noob in this and sql, need help :smile:

getting this error

`[loading] FeexRanks
[9/13/2016 1:50:42 PM] [Info] [registered] /list (Freenex.FeexRanks.CommandList/list)
[9/13/2016 1:50:42 PM] [Info] [registered] /points (Freenex.FeexRanks.CommandPoints/points)
[9/13/2016 1:50:42 PM] [Info] [registered] /rank (Freenex.FeexRanks.CommandRank/rank)
[9/13/2016 1:51:13 PM] [Error] Failed to load FeexRanks, unloading now... :System.TimeoutException: Read failure ---> System.IO.IOException: Read failure ---> System.Net.Sockets.SocketException: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond.

  at System.Net.Sockets.Socket.Receive (System.Byte[] buffer, Int32 offset, Int32 size, SocketFlags flags) [0x00000] in <filename unknown>:0 
  at System.Net.Sockets.NetworkStream.Read (System.Byte[] buffer, Int32 offset, Int32 size) [0x00000] in <filename unknown>:0 
  --- End of inner exception stack trace ---
  at System.Net.Sockets.NetworkStream.Read (System.Byte[] buffer, Int32 offset, Int32 size) [0x00000] in <filename unknown>:0 
  at MySql.Data.Common.MyNetworkStream.Read (System.Byte[] buffer, Int32 offset, Int32 count) [0x00000] in <filename unknown>:0 
  --- End of inner exception stack trace ---
  at MySql.Data.Common.MyNetworkStream.Read (System.Byte[] buffer, Int32 offset, Int32 count) [0x00000] in <filename unknown>:0 
  at MySql.Data.MySqlClient.TimedStream.Read (System.Byte[] buffer, Int32 offset, Int32 count) [0x00000] in <filename unknown>:0 
[9/13/2016 1:51:13 PM] [Info] 
[unloading] FeexRanks`

**Comment by Pustalorc** [comment]
@MysterousPotato Is the MySQL server setup in the same machine of your server, or in another machine? And if it is installed on the same machine as your unturned server, make sure that the plugin connects to the correct port running MySQL and that MySQL is running!

**Comment by MysterousPotato** [comment]
@persiafighter Ok MySQL is running, i see 2 ports here, the one of my server 25454, and this wich appear in phpMyAdmin 8089, wich one should i use?

**Comment by Pustalorc** [comment]
@MysterousPotato MySQL default's port is 3306. By what you tell me... I would say to go for 8089, but that could be another program running in that port (either a mail system or splunk [a big data analysis software]).
As I said before, if the MySQL is not hosted in the SAME machine as your server, it most likely just wont work unless you set the correct connection addresses and ports.

One last thing (and this is JUST if you are 100% SURE that MySQL is installed AND running in the SAME COMPUTER as your server is hosted on), in the config of the plugin for connecting to the database... don't set the address to the server's public address (x.x.x.x) instead just set it for localhost as that tells the computer to connect to itself.

In conclusion: try with 8089, and if it doesn't work, get the CORRECT MySQL port by going into your MySQL configuration.

**Comment by MysterousPotato** [comment]
@persiafighter it worked! thanks for your time man, i couldnt find help anywhere else xD!, everything is working fine except commands /points and /rank [own_name] :(.
But anywway, you helped me alot, thanks again. (and yes, my english is potatoed, not my native language xD)
